app/oracle.py

```diff
@@ -18,6 +18,24 @@
     level=logging.INFO, format='%(levelname)8s %(asctime)s <daemon> %(message)s', datefmt='%Y-%m-%d %H:%M:%S'
 )
 
+#### print version ####
+
+VERSION_JSON_PATH = os.environ.get('VERSION_JSON_PATH', '/version.json')
+if os.path.exists(VERSION_JSON_PATH):
+    with open(VERSION_JSON_PATH) as version_file:
+        version_info = json.load(version_file)
+else:
+    version_info = {}
+
+logging.info('version: %s' % version_info.get('version'))
+logging.info('message: %s' % version_info.get('message'))
+logging.info('commit_datetime: %s' % version_info.get('commit_datetime'))
+logging.info('build_datetime: %s' % version_info.get('build_datetime'))
+logging.info('tags: %s' % version_info.get('tags'))
+logging.info('branch: %s' % version_info.get('branch'))
+
+####
```

Comment:
Empty comments are just pollution. Functional blocks and groupings may be visualised by just blank lines